src/OrchardCore.Modules/OrchardCore.Settings/AdminMenu.cs

```diff
@@ -23,6 +23,7 @@ public void BuildNavigation(string name, NavigationBuilder builder)
             builder.Add(T["Design"], design => design
                 .Add(T["Settings"], "1", settings => settings
                     .Add(T["General"], T["General"], entry => entry
+                        .Permission(Permissions.ManageGroupSettings)
```

Comment:
There's a handful of places where permissions are missing from the menu items. I'll go through and try and add as many as I can in another PR.